Docs: Update node graph docs (#35196)
grafana/grafana: docs/sources/explore/trace-integration.md

For information on how to configure queries for the data sources listed above, refer to the documentation for specific data source.

> {{< figure src="/static/img/docs/v80/explore-trace-view-full.png" class="docs-image--no-shadow" max-width= "900px" caption="Screenshot of the trace view" >}}

Reviewer: Please put the image in the explore folder in the website repo, not v80.

Reviewer: This applies to all images. Put the images in the subject folder, not a version folder.

Author: I assume the old ones that did not change here can stay as they are or should they also be changed?

- Process metadata: Metadata about the process that logged this span.
- Logs: List of logs logged by this span and associated key values. In case of Zipkin logs section shows Zipkin annotations.

##### Node graph

> You can optionally expand node graph for the displayed trace. Depending on the data source this can show spans of the trace as nodes in the graph, or some additional context like service map based on the current trace.
> 

Reviewer 2: ```suggestion
You can optionally expand the node graph for the displayed trace. Depending on the data source, this can show spans of the trace as nodes in the graph, or as some additional context like service map based on the current trace.

```

![Node graph](/static/img/docs/v80/explore-trace-view-node-graph.png "Node graph")

##### Trace to logs

> **Note:** Available in Grafana 7.4 and later versions.

> You can navigate from a span in a trace view directly to logs relevant for that span. This is available for Tempo, Jaeger and Zipkin data source at this moment. their relevant documentation for instruction how to configure this feature.

Reviewer 2: ```suggestion
You can navigate from a span in a trace view directly to logs relevant for that span. This is available for Tempo, Jaeger, and Zipkin data sources at this moment. Refer to their relevant documentation for instructions on how to configure this feature.
```

Required fields:

| Field name   | Type                | Description                                                                                                                       |
| ------------ | ------------------- | --------------------------------------------------------------------------------------------------------------------------------- |
| traceID      | string              | Identifier for the entire trace. There should be only one trace in the data frame.                                                |
| spanID       | string              | Identifier for the current span. SpanIDs should be unique per trace.                                                              |

> | parentSpanID | string              | SpanID of the parent span to create child parent relationship in the trace view. Can be `undefined` for root span without parent. |

Reviewer 2: ```suggestion
| parentSpanID | string              | SpanID of the parent span to create child parent relationship in the trace view. Can be `undefined` for root span without a parent. |
```
